Cargo.toml

```diff
@@ -27,7 +27,7 @@ maplit = "1.0.2"
 sha2 = "0.8.1"
 serde = "1.0.116"
 serde_json = "1.0"
-serde-wasm-bindgen = "0.1.3"
+serde-wasm-bindgen = "0.3.0"
```

Comment:
This update was needed to pick a fix for internally tagged enums as well as support for `serialize_maps_as_objects(true)`